### PR TITLE
Fix/add readme

### PR DESCRIPTION
**Pull request by hugopeixoto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+In this repository you can find the style and interaction guidelines for Make
+or Break events.
+
+
+## Licensing
+
+Copyright 2019 Make or Break.
+
+The files available here are to be used only as a reference to implement the
+Make or Break website.
```
Comment (jpmcosta) [inline README.md:12]: Not sure if necessary, but a link to the project could be nice.

**Comment by jpmcosta** [comment]
Ship it.

File: README.md (hunk 1 above)
Comment (hugopeixoto) [reply README.md:12]: The github project? or the makeorbreak.io website?

**Comment by hugopeixoto** [comment]
Merging. I'll add the link in a separate PR

File: README.md (hunk 1 above)
Comment (jpmcosta) [reply README.md:12]: I was talking about the GitHub project. It would make discoverability for someone who came across this repo.

**Pull request merged**